### ISSUE TITLE
Implement services for 2018 to hacko stack.

### ISSUE THREAD

**Issue by iant01** [hackoregon/civic-devops#119]
The following PRs are ready to be merged one at a time, the relevant files copied to the S3 hacko-infrastructure-cfn  and applied to the hacko stack using the cloudformation change-set procedures by an admin with full permissions. 

Apply the changes one at a time and proceed to the next only after successfully applying the current change.

Suggested order to implement PR's:
- [x] PR 16 for transportation-systems service
- [x] PR 20 for disaster-resilience service
- [x] PR 17 for housing-affordability service
- [x] PR 23 for civic-2017 service changes
- [x] PR 21 for civic-2018 service changes
- [ ] PR 22 for 2017 service Listener sub-domain changes.
  Once project containers are in ECR
- [x] PR 18 for local-elections service
- [x] PR 19 for neighborhood-development service

Verify 2017 service API's are reachable:
- [ ] service.civicpdx.org/budget
- [ ] service.civicpdx.org/emergency
- [ ] service.civicpdx.org/endpoints
- [ ] service.civicpdx.org/homeless
- [ ] service.civicpdx.org/housing
- [ ] service.civicpdx.org/transport

Verify civic-2017 mono service endpoints are reachable:
- [ ] 2017.civicpdx.org/
- [ ] 2017.civicpdx.org/budget
- [ ] 2017.civicpdx.org/emergency
- [ ] 2017.civicpdx.org/endpoints
- [ ] 2017.civicpdx.org/homeless
- [ ] 2017.civicpdx.org/housing
- [ ] 2017.civicpdx.org/transport

Verify civic-2018 mono service endpoints are reachable:
- [ ] staging-2018.civicpdx.org/
- [ ] staging-2018.civicpdx.org/disaster
- [ ] staging-2018.civicpdx.org/elections
- [ ] staging-2018.civicpdx.org/housing
- [ ] staging-2018.civicpdx.org/neighborhood
- [ ] staging-2018.civicpdx.org/transportation

Verify 2018 API's are reachable:
- [ ] service.civicpdx.org/disaster-resilience
- [ ] service.civicpdx.org/local-elections
- [ ] service.civicpdx.org/housing-affordability
- [ ] service.civicpdx.org/neighborhood-development
- [ ] service.civicpdx.org/transportation-systems

**Comment by DingoEatingFuzz** [comment]
Everything checked above has been merged into the master branch in the infrastructure repo.

We held off on https://github.com/hackoregon/hackoregon-aws-infrastructure/pull/22 since we discovered the changes weren't critical.

**Comment by MikeTheCanuck** [comment]
With the merge of https://github.com/hackoregon/hackoregon-aws-infrastructure/pull/22, and that everything else here is live, we can close this issue.
